Source/Core/Core/HW/SystemTimers.cpp

```diff
@@ -118,7 +118,7 @@ static void IPC_HLE_UpdateCallback(u64 userdata, int cyclesLate)
 static void VICallback(u64 userdata, int cyclesLate)
 {
 	VideoInterface::Update();
-	CoreTiming::ScheduleEvent(VideoInterface::GetTicksPerHalfLine() - cyclesLate, et_VI);
+	CoreTiming::ScheduleEvent(s64(VideoInterface::GetTicksPerHalfLine()) - cyclesLate, et_VI);
```

Comment:
Should just do the change, there's not really any cost you'd avoid by keeping the one with a 32-bit parameter. cyclesLate should indeed also be made a 64-bit integer.